d.ts/mpegts.d.ts

```diff
@@ -86,6 +86,13 @@ declare namespace Mpegts {
          */
         liveBufferLatencyMinRemain?: number;
 
+        /**
+         * @desc Chasing the live stream latency caused by the internal buffer in HTMLMediaElement even if HTMLMediaElement is paused
+         *       Effective only if `isLive: true` and `liveBufferLatencyChasing: true`
+         * @defaultvalue false
+         */
+        liveBufferChasingOnPaused?: boolean;
```

Comment:
It should be better to keep the naming consistent with the prefix of this series `liveBufferLatency`.
Consider renaming it into `liveBufferLatencyChasingOnPaused` and moving to the below of `liveBufferLatencyChasing`.